tasks/xmp_i386.yml

```diff
@@ -34,7 +34,7 @@
 - name: Install i386 XMPFilesProcessor dependencies
   yum:
     name:
-      - "compat-libstdc++-33.i686"
+      - "{{ 'compat-libstdc++-33.i686' if ansible_distribution_major_version == '7' else 'libstdc++' }}"
```

Comment:
I am not sure if this is the correct solution for this. Wouldn't this also apply to Debian7 oder MyOwnLinuxOS 7?
I suggest to use the include_vars step with an appropriate configuration for this like.
* default = vars/default.yaml
* centos8 = vars/centos8.yaml

Reply:
This task comes with a `when` condition:
```
when: ansible_os_family == "RedHat"
```
So Debian and alike won't be affected.